change/@internal-react-components-f0a9c548-cc05-43c2-a29d-30b027714698.json

```diff
@@ -0,0 +1,7 @@
+{
+  "comment": "Conditional build of stable flavor",
```

Comment:
not blocking this PR, but we need all this documented well in https://github.com/Azure/communication-ui-library/tree/main/docs/infrastructure

Reply:
Oooh yes. We need that. We're not fully done here actually. @PorterNan let's together aim to document all this by the end of this sprint.